core/src/test/java/hudson/UtilTest.java

```diff
@@ -352,6 +352,7 @@ public void testDeleteContentsRecursive() throws Exception {
         assertFalse("f1 exists", f1.exists());
     }
 
+    @Ignore("TODO often fails in CI")
```

Comment:
Has long been flaky. #3787 forgot this one.

Reply:
Perhaps better to assume not running in a CI environment? https://github.com/auchenberg/volkswagen/ light?

Reply:
Already in progress for this one in [#3721](https://github.com/jenkinsci/jenkins/pull/3721/files#diff-a27cf012397023c4bd3186215bd7b2edR356). Otherwise we will have 2 @Ignore and the compilation will fail.